extensions/amp-script/validator-amp-script.protoascii

```diff
@@ -192,6 +192,12 @@ tags: {  # <amp-script>
       also_requires_attr: "script"
     }
   }
+  attrs: {
+    # This attribute should always be invalid.
+    name: "data-ampdevmode"
+    value: "false"
+    blacklisted_value_regex: "false"
```

Comment:
Nice. Probably can remove the `value` line. 

/cc @honeybadgerdontcare

Reply:
Discussed offline. Another value could be used but I don't have any issue with this specific value. The comment makes it clear that the intention is for this to be always invalid. Perhaps we could make it more explicit by using a value like `invalid-attribute` or something. I would suggest providing a link to this PR/Issue in the comment.